dmcontent/formats.py

```diff
@@ -1,15 +1,25 @@
 # -*- coding: utf-8 -*-
+from typing import Optional, Union
 
 
-def format_price(min_price, max_price, unit, interval, hours_for_price=None):
+def format_price(min_price: Optional[Union[str, float]],
+                 max_price: Optional[Union[str, float]],
+                 unit: Optional[str],
+                 interval: Optional[str],
+                 hours_for_price: Optional[str] = None):
     """Format a price string"""
+    if min_price is not None:
+        base_price = min_price
```

Comment:
This could allow the min_price to be zero, or the empty string, right?

Reply:
Yes. The empty string is obviously nonsense, though. I've just pushed an updated version that rejects the empty string. I think it also makes the logic more obvious.